tests/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '2'
+services:
+
+  test:
+    build: .
+    entrypoint: bash
+    depends_on:
+      - postgres
+      - mysql
+    volumes:
+      - ../:/project
+
+  mysql:
+    image: percona:5.7
+    environment:
+      - MYSQL_ROOT_PASSWORD=secret
+      - MYSQL_DATABASE=yiitest
+      - MYSQL_USER=travis
+      - MYSQL_PASSWORD=travis
+
+  postgres:
+    image: postgres
+    environment:
+      - POSTGRES_DB=yiitest
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+
+  #sqlsrv:
+  #  image: microsoft/mssql-server-linux
```

Comment:
Yes, I tried, but you need to install some additional packages - just did not work right away. Let me know if you've some input here, I've never worked with MSSQL.